File: domain/src/main/java/no/nav/melosys/domain/behandlingsgrunnlag/data/LuftfartBase.java
package no.nav.melosys.domain.behandlingsgrunnlag.data;

public class LuftfartBase {
    public String hjemmebaseNavn;
    public String hjemmebaseLand;
    public String typeFlyvninger;

    public LuftfartBase() {
    }

    public LuftfartBase(String hjemmebaseNavn, String hjemmebaseLand, String typeFlyvninger) {
        this.hjemmebaseNavn = hjemmebaseNavn;
        this.hjemmebaseLand = hjemmebaseLand;
        this.typeFlyvninger = typeFlyvninger;
    }
}

File: domain/src/main/kotlin/no/nav/melosys/domain/mottatteopplysninger/data/arbeidssteder/LuftfartBase.kt
package no.nav.melosys.domain.mottatteopplysninger.data.arbeidssteder

import no.nav.melosys.domain.kodeverk.Flyvningstyper


data class LuftfartBase @JvmOverloads constructor(
    var hjemmebaseNavn: String? = null,
    var hjemmebaseLand: String? = null,
    var typeFlyvninger: Flyvningstyper? = null,
    var erVanligHjemmebase: Boolean? = null,
    var vanligHjemmebaseLand: String? = null,
    var vanligHjemmebaseNavn: String? = null
)
